lib/litmus_paper/configuration_file.rb

```diff
@@ -45,12 +51,82 @@ def service(name, &block)
       @services[name.to_s] = service
     end
 
+    def yaml_service(value)
+      if value.class == Array
```

Comment:
Why is an Array of services supported in addition to a Hash?

Reply:
...I think that was actually leftover from some iteration. I'll remove that. Good catch.